js/contextMenus.js

```diff
@@ -1080,23 +1080,23 @@ function mainTemplateInit (nodeProps, frame, tab) {
             enabled: tab.get('canGoBack'),
             click: (item, focusedWindow) => {
               if (focusedWindow) {
-                CommonMenu.sendToFocusedWindow(focusedWindow, messages.SHORTCUT_ACTIVE_FRAME_BACK)
+                CommonMenu.sendToFocusedWindow(focusedWindow, [messages.SHORTCUT_ACTIVE_FRAME_BACK])
```

Comment:
odd that these are arrays. Seems like `sendToFocusedWindow` should handle `...args` and while I'm looking at it, it also seems odd that it's called `sendToFocusedWindow`, but takes a window argument. Seems like it should be `sendToWindow` or not take the `focusedWindow` as an arg